scripts/FetchMLRecs.js

```diff
@@ -0,0 +1,36 @@
+import { PrismaClient } from "@prisma/client";
+
+const prisma = new PrismaClient();
+async function fetchRatings() {
+
+        const ratings = await prisma.userMovies.findMany({
+            select: {
+                userId: true,
+                movie: {
+                    select: {
+                        movieId: true
+                    }
+                },
+                rating: true,
+            }
+        });
+        return ratings.map(r => ({
+            userID: r.userId,
+            itemID: r.movie.movieId,
+            rating: r.rating/2,
+            timestamp: r.timestamp
+        }));
+
+}
+import { spawn }from 'child_process';
```

Comment:
Typically all imports are defined at the top of the page

Reply:
Good point, fixed!